collie/model/base/base_pipeline.py

```diff
@@ -211,12 +211,19 @@ def __init__(self,
             self.hparams.num_epochs_completed = 0
 
             # ensure there are no nulls in ``item_metadata``
-            if 'item_metadata' in kwargs:
+            if 'item_metadata' in kwargs and kwargs.get('item_metadata') is not None:
```

Comment:
I think we can refactor this to only include your addition: 

```suggestion
            if kwargs.get('item_metadata') is not None:
```

Reply:
makes sense, yup!